src/bin/zmcertmgr

```diff
@@ -60,8 +60,8 @@ Where valid command/option combinations are:
   verifycrtchain <ca_chain_file> <certfile>
   verifycrtkey   <priv_key>      <certfile>
 
-  viewdeployedcrt    [all|ldap|mailboxd|mta|proxy]
-  checkcrtexpiration [all|ldap|mailboxd|mta|proxy] [-days days]
+  viewdeployedcrt    [all|ldap|mailboxd|mta|proxy|imap]
```

Comment:
At the moment, I lean towards using the name "imapd" consistently for this change as a whole instead of just "imap".  Then also use that same name for the PID/key/crt files too.  I'd like to see us use names as consistently as possible to avoid ambiguity when others have to look at this in the future.